tests/_bootstrap.php

```diff
@@ -1,5 +1,5 @@
 <?php
 // This is global bootstrap for autoloading 
-
-\Codeception\Util\Autoload::registerSuffix('Page', __DIR__.DIRECTORY_SEPARATOR.'_pages');
+\Codeception\Util\Autoload::registerSuffix('Page', __DIR__.DIRECTORY_SEPARATOR.'_pages/Joomla2/Administrator');
+\Codeception\Util\Autoload::registerSuffix('Page', __DIR__.DIRECTORY_SEPARATOR.'_pages/Joomla3/Administrator');
```

Comment:
Yeap makes sense!, that should also be a right way, I will update this, here and other places as well